app/javascript/src/admin/micro-clusters/reducer.js

```diff
@@ -138,6 +138,8 @@ const removeMicroCluster = (state, payload) => {
     state.selectedMicroClusters
   );
   if (selectedMicroClusters.length > 0) {
+    // TODO: figure out the intended behavior here
+    // eslint-disable-next-line no-self-assign
     selectedBrands = selectedBrands;
```

Comment:
This seemingly assigns `undefined` to `selectedBrands` which is already `undefined` (declared a few lines above as `let selectedBrands;`). I'm not sure what the intended behavior is here.